### PR TITLE
Metron-302 Build Common Deployment RPM

### PR DESCRIPTION
**Pull request by mmiklavc**
Create base spec for metron rpm's

https://issues.apache.org/jira/browse/METRON-302

### CODE REVIEW

File: metron-deployment/packaging/docker/rpm-docker/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+rm -rf SRPMS/ RPMS/ && \
+rpmbuild -v -ba --define "_topdir $(pwd)" --define "_version 0.2.0BETA" SPECS/metron.spec && \
+rpmlint -i SPECS/metron.spec RPMS/*/metron* SRPMS/metron
```
Comment (dlyle65535) [inline metron-deployment/packaging/docker/rpm-docker/build.sh]: Unless there's some risk that the rpmbuild command will fail silently or that the produced rpms won't work repeatable (or there's something else I'm missing), I'd drop the rpmlint command.

Comment (mmiklavc) [inline metron-deployment/packaging/docker/rpm-docker/build.sh]: rpmlint checks the spec files as well as the RPM's themselves - I'd recommend keeping it for verification that we:
1. follow RPM standards
2. don't have errors (RPM can build with error code 0, but have actual issues for installation)

File: metron-deployment/packaging/docker/rpm-docker/build.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (cestella) [inline metron-deployment/packaging/docker/rpm-docker/build.sh]: forgot the license

**Comment by mmiklavc** [comment]
Closing this PR - this will instead be included as part of another merged PR

**Pull request closed without merging**